Source/WebKit/UIProcess/InspectorServer/wpe/WebInspectorServerWPE.cpp

```diff
@@ -44,7 +44,7 @@ static void loadResourcesIfNeeded()
 {
     static std::once_flag flag;
     std::call_once(flag, [] {
-        GModule* resourcesModule = g_module_open("libWPEWebInspectorResources.so", G_MODULE_BIND_LAZY);
+        GModule* resourcesModule = g_module_open(PKGLIBDIR G_DIR_SEPARATOR_S "libWPEWebInspectorResources.so", G_MODULE_BIND_LAZY);
```

Comment:
Hi @neeraj-deshpande can you tell me why this is necessary? Does it not work otherwise?

Reply:
Hi @MFransen69 
libWPEWebInspectorResources.so doesn’t reside in /usr/lib.

For WebKit 2.22, it resides in /usr/lib/wpe-webkit-0.1/.
For WebKit 2.28 resides in /usr/lib/wpe-webkit-1.0/.

WebKit 2.28 also loads libWPEWebInspectorResources.so similarily here:
https://github.com/WebPlatformForEmbedded/WPEWebKit/blob/b2e6a99cdbd58952a56a45c70490704fb5c7cd45/Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorUtils.cpp#L44